src/components/Pokemon.js

```diff
@@ -0,0 +1,50 @@
+import React, {useEffect, useState} from 'react';
+import axios from "axios";
+
+function Pokemon({pokemon}) {
+
+    const [pokemonData, setPokemonData] = useState({});
+    const [error, setError] = useState(null);
+
+    useEffect(() => {
+        async function fetchData() {
+            setError('');
```

Comment:
Ik zou deze `null` of `undefined` maken. Dat is iets netter dan een lege string namelijk